core/backend/package.json

```diff
@@ -84,7 +84,7 @@
     "null-loader": "^4.0.1",
     "nyc": "^15.1.0",
     "rimraf": "^3.0.2",
-    "sinon": "^9.0.2",
+    "sinon": "^15.0.4",
```

Comment:
@types/sinon doesnt need to be updated??

Reply:
I don't think it is necessary for our usecase, the latest they have is 10 for types. What's a good way to determine if update to types is needed?

Reply:
Typically when you update a pkg (especially for new major), and they doesnt deliver their own type, and there is a corresponding `@types/` pkg, youll want to update those as well. Typically there should be a new major types ver for every major of the pkg.

Id recommend going through the changelog for sinon to understand the changes: https://github.com/sinonjs/sinon/blob/main/CHANGES.md

Also take a look at the DefinitelyTyped repo to understand how this works
Based off https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sinon/index.d.ts#L1, this ws meant for Sinon 10, but i guess their api hasnt changed much since so it should be safe to use that

Take a look at presentation thats using sinon@15 & types/sinon@10 https://github.com/iTwin/presentation/blob/master/packages/components/package.json

Reply:
Sounds good, thanks for the tips!